bus-mapping/src/circuit_input_builder/block.rs

```diff
@@ -23,6 +23,8 @@ pub struct BlockContext {
     pub(crate) call_map: HashMap<usize, (usize, usize)>,
     /// Total gas used by previous transactions in this block.
     pub(crate) cumulative_gas_used: u64,
+    /// Number of invalid transactions
+    pub(crate) num_invalid_tx: usize,
```

Comment:
Didn't find the place derive logic from this new defined field. is it left over?

Reply:
Yes, left it in but indeed better to remove for now.